src/app/helptext/system/failover.ts

```diff
@@ -23,17 +24,24 @@ export const helptext_system_failover = {
     sync_to_message: T("Sync to peer succeeded."),
   },
 
-  disabled_placeholder: T('Disabled'),
-  disabled_tooltip: T(''),
+  disabled_placeholder: T('Disable Failover'),
+  disabled_tooltip: T('Disable automatic failover.'),
 
-  master_placeholder: T('Master'),
-  master_tooltip: T(''),
+  master_placeholder: T(`Default ${globalHelptext.ctrlr}`),
+  master_tooltip: T(`Mark the currently active ${globalHelptext.ctrlr} \
```

Comment:
"Mark" is still a weird explanation of this.  It does not mark the controller, it makes it the active one.
```suggestion
  master_tooltip: T(`Make the currently active ${globalHelptext.ctrlr} \
```